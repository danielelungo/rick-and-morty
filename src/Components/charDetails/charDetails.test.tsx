import React from "react";
import renderer from "react-test-renderer";
import CharDetail from "./charDetails";

describe("CharDetails component", () => {
  test("renders CharDetails Component", () => {
    const tree = renderer
      .create(<CharDetail data={10} list={false} title={"origin"} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
