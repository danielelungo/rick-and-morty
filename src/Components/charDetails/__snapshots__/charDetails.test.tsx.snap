// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharDetails component renders CharDetails Component 1`] = `
<div
  className="sc-bczRLJ gVHrnU"
>
  <h4
    className="sc-dkzDqf"
  >
    origin
     
    10
  </h4>
</div>
`;
